### ISSUE TITLE
Rails 5 Deprecation warning - alias_method_chain

### ISSUE THREAD

**Issue by shadowed87** [lml/commontator#105]
After upgrading to Rails 5 I am receiving the following deprecation warning:

`
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from block in acts_as_commontable at /usr/lib/ruby/gems/2.3.0/gems/commontator-4.11.1/lib/commontator/acts_as_commontable.rb:32)
`

**Comment by Dantemss** [comment]
Thanks. I will probably switch to using `alias` instead to keep compatibility with old versions.

**Comment by niborg** [comment]
Any status with this? Or you looking for a PR?

**Comment by chao-mu** [comment]
So glad there is a fix for this and that it's been merged.

Is there a release on the horizon with this fix such that this can be pulled in via a gemspec in an engine? I'd like to use this in a work project, but am worried about selling it to my coworkers if there's not a released version that works for us.

**Comment by Dantemss** [comment]
Would you be willing to test the release? I am not using the gem in any project currently, so any help testing would be appreciated.

**Comment by chao-mu** [comment]
Unfortunately this turned out not to be a good fit for our project. Thank you for your time!

For what it's worth, the other bug I noticed in testing was that migrations don't specify version explicitly, making them incompatible with newer rails versions.

**Comment by Dantemss** [comment]
Migration issue fixed in https://github.com/lml/commontator/pull/121

**Comment by mulander** [comment]
@Dantemss I would be willing to help test a release, the released gem doesn't work with Rails 5. I'm starting a new project using this gem and currently got it working by pointing at the latest commit in this git repository.

**Comment by ortegacmanuel** [comment]
We just did the same - point at the master branch - on our project in order to get it working. We also can help with release testing.

**Comment by mulander** [comment]
@ortegacmanuel since that comment I helped test a new release which was published 27 days ago as `v5.0.0`.

The README has not been updated to reflect that (I'll send a pull request for that shortly)
just change 

```ruby
gem 'commontator', '~> 4.11.1'
```

to
```ruby
gem 'commontator', '~> 5.0.0'
```

in your Gemfile.

**Comment by Dantemss** [comment]
Yep, thanks to @mulander's help I have released version 5.0.0 (and now 5.1.0) which fix this issue.
